plugins/soundsourcem4a/soundsourcem4a.cpp

```diff
@@ -126,6 +126,11 @@ Result SoundSourceM4A::tryOpen(const AudioSourceConfig& audioSrcCfg) {
         return ERR;
     }
 
+    if (MP4GetTrackFixedSampleDuration(m_hFile, m_trackId) != kFramesPerSampleBlock) {
```

Comment:
Not sure what you mean by "put a TODO" but I just created a Launchpad bug: https://bugs.launchpad.net/mixxx/+bug/1473690